components/cli/pkg/commands/image/test.go

```diff
@@ -598,107 +474,98 @@ func SetupDebugConfigs(iName string, verboseMode string, imageDir string, depend
 	content = append(content, fmt.Sprintf("DEPENDENCY_LINKS=\"%s\"\n",
 		strings.Replace(dependencyLinks, "\"", "\\\"", -1)))
 
-	ballerinaConf := filepath.Join(util.UserHomeCelleryDir(), constants.TMP, constants.BallerinaConf)
-	isExistsBalConf, err := util.FileExists(ballerinaConf)
+	ballerinaConfPath := filepath.Join(balProj, constants.BallerinaConf)
+
+	isExistsBalConf, err := util.FileExists(ballerinaConfPath)
 	if err != nil {
-		util.ExitWithErrorMessage("error while checking if "+ballerinaConf+" exists", err)
+		return err
 	}
 	if isExistsBalConf {
-		err := os.Remove(ballerinaConf)
+		err := os.Remove(ballerinaConfPath)
 		if err != nil {
-			util.ExitWithErrorMessage("error while removing "+ballerinaConf+" file", err)
+			return err
 		}
 	}
-	_, err = os.Create(ballerinaConf)
+	_, err = os.Create(ballerinaConfPath)
 	if err != nil {
-		util.ExitWithErrorMessage("error while creating "+ballerinaConf+" file", err)
+		return err
 	}
 
-	f, err := os.OpenFile(ballerinaConf, os.O_APPEND|os.O_WRONLY, 0600)
+	f, err := os.OpenFile(ballerinaConfPath, os.O_APPEND|os.O_WRONLY, 0600)
```

Comment:
Can't we use `O_CREATE` rather using `os.Create(ballerinaConfPath)` seperately?

Reply:
Fixed